packages/@aws-cdk/aws-synthetics/README.md

```diff
@@ -9,8 +10,176 @@
+	handler: 'index.handler',
+	code: Code.fromInline('your code here'),
+});
 ```
+
+### A Note about Canaries
+
+This canary construct is a representation of AWS Cloudwatch Synthetic Canaries. There is such a thing as **canary deployment** available in AWS Lambda, AWS CodeDeploy, etc. AWS Cloudwatch Synthetic Canaries are not that and a different resource entirely.
+
+### Canary Properties
+
+Here is how the Canary Cloudformation properties match onto the CDK Canary construct.
+
+#### Cloudformation Required Properties: 
+
+`artifactS3Location`: Specify the s3 bucket location that the canary will store its data in. Optional for the Canary construct which defaults to creating a new s3 bucket for you.
+ 
+`code` : Specify the canary script, either as an object with `handler` and `script` properties or `handler`, `s3bucket`, `s3key`, and `s3version?`. The Canary construct splits this property into two required properties: `handler` and `code`. As such, it closely resembles [properties](https://github.com/aws/aws-cdk/tree/3256a41787c365a67b01bee193bd75e48645f7a0/packages/@aws-cdk/aws-lambda) in the Lambda constructor. 
+	
+ - It is important to point out that, unlike Lambda, the canary `handler` must be a string that ends in `.handler`.
+
+`executionRoleArn`: Specify the arn of the role that the canary will use to run its script. The Canary construct replaces this property in favor of an optional `role: iam.IRole`. This is convention amongst other constructs like Lambda. If left unspecified, the construct will create a IAM role with the necessary permissions to perform the canary. If specified, it must have a principal of `lambda.amazonaws.com` and have these permissions attached:
+-   `s3:PutObject`
+-   `s3:GetBucketLocation`
+-   `s3:ListAllMyBuckets`
+-   `cloudwatch:PutMetricData`
+-   `logs:CreateLogGroup`
+-   `logs:CreateLogStream`
+-   `logs:PutLogEvents` 
+
+`name` : The name of the canary. Must be lowercase and have no spaces. Redefined as `canaryName` in the Canary construct.
+
+`runConfig` : The construct that contains imput about each canary run. The only property is `timeoutInSeconds`. The Canary construct replaces `runConfig` with the optional`timeout` property that defaults to the `rate` of the canary.
+
+`runtimeVersion`: Specify the runtime version of the canary. The only valid runtime is `syn-1.0`. The Canary construct avoids future proofing and does not expose this property at all -- `syn-1.0` is hardcoded as the `runtimeVersion`. If more runtime versions are possible in the future, this can be changed to an enum-like property.
+
+`schedule` : Specify how often the canary is to run and when the runs should stop. Type is a schedule object with two properties, `durationInSeconds` and `expression`. The Canary construct just exposes both as optional properties:
+-  `lifetime` replaces `durationInSeconds` and defaults to `0` (canary runs forever until you stop it).
+- `rate` replaces `expression` and defaults to `rate(5 minutes)`. 
```

Comment:
what about changing `rate` to `frequency`? It seems like this property encapsulates how frequent the canary runs.

Reply:
this has been changed to `Schedule` to fit similar properties in autoscaling constructs.